packages/react-dom/src/__tests__/ReactDOMProduction-test.js

```diff
@@ -211,6 +211,28 @@ describe('ReactDOMProduction', () => {
     );
   });
 
+  // Regression test verifying that trying to access (missing) component stack doesn't crash.
+  it('should throw on children for void elements', () => {
+    const errorCode = 137;
```

Comment:
You could put the entire error string into this var since it's duplicated below?

Reply:
Yeah, I guess.. Just copy paste from above tests. I don't really care. I'll have to get back to this and find another way to test it when I enable production testing of bundles.

Reply:
Sure, no big deal. Just seemed pretty arbitrary the way it was currently setup.

Reply:
I agree. I just don't want to spend time here because I'll delete those tests anyway.